frontend/app/src/components/dashboard/edit-snapshot/EditBalancesSnapshotLocationSelector.vue

```diff
@@ -25,7 +25,7 @@ const { t } = useI18n();
 </script>
 
 <template>
-  <VSheet outlined class="pa-4" rounded>
+  <RuiCard variant="outlined" class="pa-4" rounded="sm" no-padding>
```

Comment:
```
<RuiCard variant="outlined" rounded="sm" class="[&>div]:!overflow-visible">
```

Reply:
Use this, if not, the selection will be clipped.
<img width="892" alt="image" src="https://github.com/rotki/rotki/assets/26648140/e112f3c7-2a38-473a-981d-6f4b99d50aed">